README.md

```diff
@@ -51,17 +51,41 @@ contain alphanumeric characters, `-` and `_`:
 pusher.trigger(u'a_channel', u'an_event', {u'some': u'data'})
 ```
 
-If you wish to trigger an event on multiple channels, pass in a list as the first parameter:
+Triggering Events
+-----------------
 
-```python
-pusher.trigger([u'a_channel', u'another_channel'], u'an_event', {u'some': u'data'})
-```
+To trigger an event on one or more channels, use the `trigger` method on the `Pusher` object.
+
+##### Event Buffer
```

Comment:
How do I for each section first then details. We're going into the Event Buffer stuff too soon here. See:

* https://github.com/pusher/pusher-http-php/tree/event-buffer#event-buffer
* https://github.com/pusher/pusher-http-dotnet/tree/event-buffer#event-buffer